### PR TITLE
Using HttpAlias to cover the Multilingual and site group scenario

### LINKED ISSUE #2308: Wrong Redirect after login for multilingual site
## Description of bug
Wrong Redirect after login for multilingual site

## Steps to reproduce
Redirect After Login Fails (Settings ->Security ->Basic Login Setting -> Redirect After Login)
No errors reported in Event Viewer.
Two language site.
It should redirect to our Statistics page.
The Statistics page is still appears set in the "Redirect After Login;"
The Table Number appears correct in dbo.PortalSeetings->Redirect_AfterLogin

## Current result
Currently it is redirecting to home page

## Expected result
It should redirect to statistics page

## Affected version
* [x] 9.2.2
* [x] 9.2.1

### PR DESCRIPTION
**Pull request by Mohtshum**
Fixes #2308 

## Summary
Using HttpAlias to cover the Multilingual and site group scenario

### CODE REVIEW

**Comment by skamphuis** [comment]
I believe this fixes an issue that is a duplicate from issue #2267 . It is fixed in pullrequest #2275

**Comment by Mohtshum** [comment]
The PR #2275 might fail for site group with multilingual site.

**Comment by skamphuis** [comment]
Right. I'm on platform so I'm not that familiar with site groups. Could you describe how I could reproduce the issue in that case?

File: Website/DesktopModules/Admin/Authentication/Login.ascx.cs [hunk 1]
```diff
@@ -199,15 +202,18 @@ protected string RedirectURL
...
-                var comparison = StringComparison.InvariantCultureIgnoreCase;
-                var isDefaultPage = redirectURL == "/"
-                    || (alias.Contains("/") && redirectURL.Equals(alias.Substring(alias.IndexOf("/", comparison)), comparison));
-                
+                var aliasWithoutDomain = alias.Substring(Request.Url.Host.Length);
+                var redirectURLTrimmed = RemoveQueryStrings(redirectURL).TrimEnd(SLASH.ToCharArray());
+
+                var isDefaultPage = redirectURL == SLASH ||
+                    (aliasWithoutDomain.Contains(SLASH)
+                        && aliasWithoutDomain.Contains(redirectURLTrimmed));
```
Comment (bdukes) [inline Website/DesktopModules/Admin/Authentication/Login.ascx.cs]: These `Contains` checks should probably still be case-insensitive, right?

**Comment by Mohtshum** [comment]
@skamphuis, Sorry, I overlooked your changes. Your PR is working perfectly well with Multilingual child sites as well.

**Pull request closed without merging**